FullDemo/app/src/main/java/com/kaltura/playkit/samples/fulldemo/PlaybackControlsView.java

```diff
@@ -74,79 +79,137 @@ private void initPlaybackControls() {
         btnNext.setOnClickListener(this);
         btnPrevious.setOnClickListener(this);
 
-        seekBar = this.findViewById(R.id.mediacontroller_progress);
-        seekBar.setOnSeekBarChangeListener(this);
+        seekBar = this.findViewById(R.id.kexo_progress);
+        seekBar.setPlayedColor(ContextCompat.getColor(getContext(), R.color.colorAccent));
+        seekBar.setBufferedColor(ContextCompat.getColor(getContext(), R.color.grey));
+        seekBar.setUnplayedColor(ContextCompat.getColor(getContext(), R.color.black));
+        seekBar.setScrubberColor(ContextCompat.getColor(getContext(), R.color.colorAccent));
+        seekBar.addListener(componentListener);
 
-        tvCurTime = this.findViewById(R.id.time_current);
-        tvTime = this.findViewById(R.id.time);
+        tvCurTime = this.findViewById(R.id.kexo_position);
+        tvTime = this.findViewById(R.id.kexo_duration);
     }
 
+
     private void updateProgress() {
-        long duration = Consts.TIME_UNSET;
-        long position = Consts.POSITION_UNSET;
+        long duration = C.TIME_UNSET;
```

Comment:
why this was changed?

Reply:
FullDemo was still using Seekbar so to use DefaultTimeBar, I have replaced the whole class.